hasher-matcher-actioner/hmalib/hashing/__init__.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
```

Comment:
Nit/YMMV:  I don't think we've been doing stuff in the `__init__` files, can sometimes make it hard to find the right file. I have a small preference for just using named files.